src/constants.ts

```diff
@@ -1,4 +1,10 @@
 export const NODE_ENV = process.env.NODE_ENV;
-export const ENDPOINT = NODE_ENV === 'prod' ?
+export const AINFT_SERVER_ENDPOINT = NODE_ENV === 'prod' ?
   'https://ainft-api.ainetwork.ai':
   'https://ainft-api-dev.ainetwork.ai';
+
+export const BLOCKCHAIN_ENDPOINT = NODE_ENV === 'prod'
```

Comment:
what about AIN_BLOCKCHAIN_ENDPOINT for the case it has other blockchains?

Reply:
Done!